composer.json

```diff
@@ -52,7 +52,10 @@
     "autoload": {
         "psr-4": {
             "JMS\\Serializer\\": "src/"
-        }
+        },
+        "files": [
+            "src/Functions/iterable_to_array.php"
```

Comment:
Functions can not be auto-loaded, this means that this file will be read on each request. 
You can place this utility function in a static method inside some class, and autoloading will work

Reply:
I tried Functions, WDYT?

Reply:
im good with it